src/components/v5/shared/Avatar/Avatar.tsx

```diff
@@ -46,7 +48,16 @@ const Avatar: FC<PropsWithChildren<AvatarProps>> = ({
       title={title}
     >
       {source ? (
-        <div className={styles.image} style={imageStyle} />
+        <div
+          className={clsx(styles.image, 'rounded-lg border-2', {
```

Comment:
Please move styles here. Isn't it broken if there is no mode added and you are adding border-2?